ipwb/replay.py

```diff
@@ -775,7 +773,7 @@ def isUri(str):
 
 def generateNoMementosInterface_noDatetime(urir):
     msg = '<h1>ERROR 404</h1>'
-    msg += 'No capture(s) found for {0}.'.format(urir)
+    msg += f'No capture(s) found for {urir}.'
```

Comment:
* Adding paragraph tag as we have head tag in the previous line
* The plural form `captures` should be good enough, no need to have parenthetical notation

```suggestion
    msg += f'<p>No captures found for {urir}.</p>'
```